cockroachcloud/cmek.md

```diff
@@ -50,7 +50,7 @@ CMEK helps you to enforce such business rules on {{ site.data.products.db }} clu
 
 ## How CMEK works
 
-When you create a {{ site.data.products.dedicated }} cluster, its data at rest is not encrypted by default. Instead, the data is stored on disks attached to the cluster, which are always encrypted by the Information-As-A-Service (IAAS) provider you select when you create the cluster.
+When you create a {{ site.data.products.dedicated }} cluster, its data at rest on cluster disks is not encrypted by default. However, the disks themselves are automatically encrypted by cryptographic keys owned and managed by the cloud providers themselves.
```

Comment:
Sorry maybe it's how it shows up on my diff, but is this para not duplicated? Please ignore if it's not.

Reply:
Sorry, I don't see duplication in the diff or in the file. Am I missing something?